packages/cms-lib/hubdb.js

```diff
@@ -8,9 +8,10 @@ const {
   fetchTable,
   fetchRows,
   publishTable,
+  deleteRows,
 } = require('./api/hubdb');
 
-async function createHubDbTable(portalId, src) {
+function validateJson(src) {
```

Comment:
I think that this function should have file in its name since it is validating that it is a readable JSON file not just that it is JSON.